rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -255,339 +473,536 @@ def resize(*inputs, bytes_per_sample_hint=0, image_type=0, interp_type=1, mag_fi
     else:
         crop_depth = crop_d
         crop_height = crop_h
         crop_width = crop_w
-    #Set Seed
-    b.setSeed(seed)
+
     if ((crop_width == 0) and (crop_height == 0)):
         # pybind call arguments
-        kwargs_pybind = {"input_image0": inputs[0],"is_output": False, "crop_width":None, "crop_height":None, "crop_depth":None ,"crop_pos_x": None, "crop_pos_y": None, "crop_pos_z": None }
-        cropped_image = b.Crop(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+        kwargs_pybind = {"input_image": inputs[0], "crop_width": None, "crop_height": None, "crop_depth": None, "is_output": False, "crop_pos_x": None,
+                         "crop_pos_y": None, "crop_pos_z": None, "rocal_tensor_output_layout": rocal_tensor_output_layout, "rocal_tensor_output_datatype": rocal_tensor_output_datatype}
+        cropped_image = b.crop(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     else:
         # pybind call arguments
-        kwargs_pybind = {"input_image0": inputs[0], "crop_width": crop_width, "crop_height": crop_height, "crop_depth": crop_depth ,"is_output": False,"crop_pos_x": crop_pos_x, "crop_pos_y": crop_pos_y, "crop_pos_z": crop_pos_z }
-        cropped_image = b.CropFixed(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+        kwargs_pybind = {"input_image": inputs[0], "crop_width": crop_width, "crop_height": crop_height, "crop_depth": crop_depth, "is_output": False, "crop_pos_x": crop_pos_x,
+                         "crop_pos_y": crop_pos_y, "crop_pos_z": crop_pos_z, "rocal_tensor_output_layout": rocal_tensor_output_layout, "rocal_tensor_output_datatype": rocal_tensor_output_datatype}
+        cropped_image = b.cropFixed(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     return (cropped_image)
 
-def color_twist(*inputs, brightness=1.0, bytes_per_sample_hint=0, contrast=1.0, hue=0.0, image_type=0,
-                preserve=False, saturation=1.0, seed=-1, output_layout=types.NHWC, output_dtype=types.UINT8, device=None):
-    brightness = b.CreateFloatParameter(brightness) if isinstance(
-        brightness, float) else brightness
-    contrast = b.CreateFloatParameter(
-        contrast) if isinstance(contrast, float) else contrast
-    hue = b.CreateFloatParameter(hue) if isinstance(hue, float) else hue
-    saturation = b.CreateFloatParameter(saturation) if isinstance(
-        saturation, float) else saturation
-    # pybind call arguments
-    kwargs_pybind = {"input_image0": inputs[0], "output_layout": output_layout, "output_dtype": output_dtype, "is_output": False,
-                     "alpha": brightness, "beta": contrast, "hue": hue, "sat": saturation}
-    color_twist_image = b.ColorTwist(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+
+def color_twist(*inputs, brightness=1.0, contrast=1.0, hue=0.0,
+                saturation=1.0, rocal_tensor_output_layout=types.NHWC, rocal_tensor_output_datatype=types.UINT8):
```

Comment:
why do we changes the name to rocal_tensor_xxx and rocal_tensor_output_xx etc. The previous name is more appropriates

Reply:
Changed the names to output_layout and output_dtype